client/src/Context.jsx

```diff
@@ -1,5 +1,5 @@
 import io from 'socket.io-client';
 import { createContext } from 'react';
 
-export const socket = io.connect(import.meta.env.VITE_SOCKET_URL);
+export const socket = io.connect('http://localhost:4000');
```

Comment:
you should leave this to the env file, because it's the same thing over there

you didn't have to delete the env file, you can just change it locally and don't push it

Reply:
Just now fixed this as well.